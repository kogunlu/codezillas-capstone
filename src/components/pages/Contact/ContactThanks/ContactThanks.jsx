import Thanks from "../../../shared/thanksPages/Thanks"

function ContactThanks(){


    return(
            <div>
                <Thanks text="Your request has been sent, a member of the support team will get in contact with you through the email you provided as soon as possible."/>
            </div>
    )
}

export default ContactThanks