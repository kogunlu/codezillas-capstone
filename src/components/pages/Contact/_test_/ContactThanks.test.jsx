import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer";
import ContactThanks from "../ContactThanks/ContactThanks"


it('Renders correctly for ContactThanks Component', () => {
    const tree = renderer
    .create(
    <BrowserRouter>
    <ContactThanks />
    </BrowserRouter>)
    .toJSON();
    expect(tree).toMatchSnapshot();
  }); 
