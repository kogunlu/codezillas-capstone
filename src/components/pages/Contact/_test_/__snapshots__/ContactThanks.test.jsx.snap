// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly for ContactThanks Component 1`] = `
<div>
  <div
    className="p-10 h-72 md:h-64"
  >
    <div
      className="h-full w-full flex flex-col justify-between"
    >
      <h2
        className="text-4xl font-medium"
      >
        THANK YOU!
      </h2>
      <p
        className="text-gray-400 text-lg font-medium"
      >
        Your request has been sent, a member of the support team will get in contact with you through the email you provided as soon as possible.
      </p>
      <div
        className="w-4/6 md:w-3/6 xl:w-2/6"
      >
        <input
          className="py-2 px-5 w-10/12 md:w-8/12 lg:w-5/12 hover:bg-cyan-200 bg-cyan-400 font-medium rounded text-sm md:text-lg"
          onClick={[Function]}
          type="button"
          value="BACK TO HOME"
        />
      </div>
    </div>
  </div>
</div>
`;
